package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Conditional expresstion util for server/client",
   "main": "./dist/index.js",
   "scripts": {
-    "test": "gulp &&./node_modules/.bin/mocha --reporter spec",
-    "prepublish": "gulp"
+    "test": "mocha --compilers js:babel-core/register --reporter spec",
+    "build": "babel src --out-dir dist",
```

Comment:
You can use `babel-cli` directly, and avoid the unneeded layer of `gulp` in this case.